pillar.example

```diff
@@ -37,15 +37,15 @@ salt:
     salt_ssh: 'salt-ssh'
     pyinotify: 'python-pyinotify'  # the package to be installed for pyinotify
 
-  # Set which salt repository to use, default to https://repo.saltproject.io
-  # For older releases use https://archive.repo.saltproject.io
-  repo: 'https://archive.repo.saltproject.io'
+  # Set which salt repository to use, default to https://packages.broadcom.com/artifactory
+  repo: 'https://packages.broadcom.com/artifactory'
+  repo_key_url: 'https://packages.broadcom.com/artifactory/api/security/keypair/SaltProjectKey/public'
```

Comment:
This should fix the failure in the pipeline for the pre-commit job https://gitlab.com/saltstack-formulas/salt-formula/-/jobs/8356162035

```suggestion
  # yamllint disable rule:line-length
  repo: 'https://packages.broadcom.com/artifactory'
  repo_key_url: 'https://packages.broadcom.com/artifactory/api/security/keypair/SaltProjectKey/public'
  # yamllint enable rule:line-length
```